index.html

```diff
@@ -5477,10 +5593,24 @@ <h3>Algorithm</h3>
                   <code>xsd:double</code> and its
                   <a>lexical form</a>
                   is a valid <code>xsd:integer</code> or <code>xsd:double</code>
-                  according [[XMLSCHEMA11-2]], set <var>converted value</var>
+                  according to [[XMLSCHEMA11-2]], and if it is a numeric value
+                  conformant to [[IEEE-754-2008]], then set <var>converted value</var>
                   to the result of converting the
                   <a>lexical form</a>
-                  to a JSON <a>number</a>.</li>
+                  to a JSON <a>number</a>.
+                </li>
+                <li>
+                  Otherwise, if <a>lexical form</a> expresses a numerical value
+                  outside of the range mandated by [[IEEE-754-2008]], then set
```

Comment:
There are several ranges supported by [IEE-754-2008 (and IEE-754-2019, the latest rev, to which we might want to update our references)](https://en.wikipedia.org/wiki/IEEE_754), depending on the subtype of the numeric.
```suggestion
                  Otherwise, if <a>lexical form</a> expresses a numeric value
                  outside of the ranges supported by [[IEEE-754-2008]], then set
```